app/api/sessions.py

```diff
@@ -57,6 +58,15 @@ def after_create_object(self, session, data, view_kwargs):
             send_email_new_session(organizer_email, event_name, link)
             send_notif_new_session_organizer(organizer, event_name, link, session.id)
 
+        if session.speakers != []:
```

Comment:
Not a good way to check for empty list

Reply:
BTW this is a redundant list

Reply:
@iamareebjamal A `len(session.speakers) != 0` would be fine?

Reply:
No need. Directly use the loop

Reply:
understood!